### PR TITLE
[ES] Update `Users with unique titles` and clean up the French translation

### PR DESCRIPTION
**Pull request by agatemosu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/People/Users_with_unique_titles/en.md [hunk 1]
```diff
@@ -354,7 +354,7 @@ These users were awarded their respective titles for miscellanous reasons.
...
 [^billiummoto-note]: ::{ flag=US }:: [BilliumMoto](https://osu.ppy.sh/users/3862471) was awarded with the **osu!mixer II** title instead for having won both the [osu!remix contest #1](https://osu.ppy.sh/home/news/2016-08-29-osuremix-contest-1-results) and the [A Labour of Love](/wiki/Contests/A_Labour_of_Love_Contest) contest.
 [^yf-note]: ::{ flag=CN }:: [yf_bmp](https://osu.ppy.sh/users/1243669) was awarded with the **Elite Storyboarder II** title instead for having [already received the title](https://osu.ppy.sh/home/news/2021-03-19-community-contributors-2020) due to outstanding contributions to the storyboarding scene, as well as having won the [triangles Beatmapping Contest](/wiki/Contests/triangles_Beatmapping_Contest).
-[^elite-note]: ::{ flag=TN }:: [Hivie](https://osu.ppy.sh/users/14102976) and ::{ flag=US }:: [Secre](https://osu.ppy.sh/users/2306637) were awarded with the **Elite Nominator II** title instead for having been named as Elite Nominators for two years in a row.
```
Comment (Walavouchey) [inline wiki/People/Users_with_unique_titles/en.md:360]: this change is unnecessary btw

Comment (agatemosu) [reply wiki/People/Users_with_unique_titles/en.md:360]: I know, but I couldn't find the note when I saw the file

**Review by TicClick — APPROVED** [review]
- FR: looks good to me
- DE/EN: pointless imo, but not worth mulling over

**Comment by TicClick** [comment]
Juan approves: https://discord.com/channels/188630481301012481/218677502141399041/1083911092352331848

**Pull request merged**